pyramid/paster.py

```diff
@@ -53,7 +53,7 @@ def get_appsettings(config_uri, name=None, options=None, appconfig=appconfig):
         global_conf=options)
 
 def setup_logging(config_uri, fileConfig=fileConfig,
-                  configparser=configparser):
+                  configparser=configparser, global_conf=None):
```

Comment:
If you look at `docs/api/paster.rst` you will see that `fileConfig` and `configparser` are private apis used for testing. This option should be documented and go ahead of the private options.